graph_objects/layout_gen.go

```diff
@@ -42,9 +42,9 @@ type Layout struct {
 	Bargroupgap float64 `json:"bargroupgap,omitempty"`
 
 	// Barmode
-	// default: stack
+	// default: group
 	// type: enumerated
-	// Determines how bars at the same location coordinate are displayed on the graph. With *stack*, the bars are stacked on top of one another With *overlay*, the bars are plotted over one another, you might need to an *opacity* to see multiple bars.
+	// Determines how bars at the same location coordinate are displayed on the graph. With *stack*, the bars are stacked on top of one another With *relative*, the bars are stacked on top of one another, with negative values below the axis, positive values above With *group*, the bars are plotted next to one another centered around the shared location. With *overlay*, the bars are plotted over one another, you might need to an *opacity* to see multiple bars.
```

Comment:
How did you change this file? it should be generated automatically from the schema and I don't see schema changes on this PR

Reply:
Made a mistake somewhere. The schema update didnt actually work at the end so the [update: ploty to 2.18.1](https://github.com/MetalBlueberry/go-plotly/pull/10/commits/1d8c908fb6c06eb36f364f146610ed1d24e79b79) commit does nothing really. When I tried getting latest schema today it failed generation. I need to look into this a bit further.